app/src/main/java/org/mozilla/focus/browser/WebViewCache.kt

```diff
@@ -42,4 +42,9 @@ class WebViewCache {
         cachedView?.removeFromParentIfAble()
         return cachedView ?: createAndCacheEngineView()
     }
+
+    fun clear() {
```

Comment:
Two ideas:
- I wonder if this should be called `onDestroy` to make it clear that it should be called in `onDestroy`
- If we're doing ^, then maybe we should use a lifecycle observer to make it automatic

*Then again*, a `WebViewCache` can be used outside the context of an Activity, in theory (though, it is dealing with views whose lifecycles are the same as contexts...) so maybe it doesn't make sense to name it as if the lifecycle is the same.